src/exec_cmd.rs

```diff
@@ -10,7 +10,7 @@ pub fn run(cmd: ExecCmd) -> Result<(), Error> {
         .map(|d| PathBuf::from(d))
         .unwrap_or(get_current_dir_or_exit());
 
-    if let Some(shadowenv) = hook::load_env(pathbuf, data, true)? {
+    if let Some(shadowenv) = hook::load_env(pathbuf, data, true, false)? {
```

Comment:
outside the scope of this PR likely but with 4 args, 2 of which are bools, i'd make a struct for these so it's clearer what's going on